spec/requests/posts_spec.rb

```diff
@@ -2,16 +2,24 @@
 
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    it 'returns http success' do
-      get '/posts/index'
+    it 'returns http success for Posts index page' do
+      get '/users/:user_id/posts'
       expect(response).to have_http_status(:success)
     end
+    it 'renders the index template for posts' do
+      get '/users/:user_id/posts'
+      expect(response.body).to include('Show posts related to user ID')
+    end
   end
```

Comment:
- [ ] Good job setting up the tests. Kindly add one more test to each of the tested actions to test for the correct rendered template. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20a%20correct%20template%20was%20rendered.) requirement.

Reply:
@ShadyShawkat Thanks bro! The changes asked by you has been done 😃